test/expect-tests/dune_rpc_e2e/dune_rpc_e2e.ml

```diff
@@ -180,4 +180,5 @@ let run run =
     ~finally:(fun () -> Sys.chdir cwd)
     ~f:(fun () ->
       Sys.chdir (Path.to_string dir);
-      Scheduler.Run.go config run ~timeout:5.0 ~on_event:(fun _ _ -> ()))
+      Scheduler.Run.go ~file_watcher:Scheduler.Run.No_watcher config run
```

Comment:
Shall we make the `file_watcher` mandatory? Btw, I'm guessing the compiler should be fine with just `~file_watcher:No_watcher` here (i.e. without the full module qualification).

Reply:
yeah, it's worth considering. On the other hand, we can also factor the scheduler a little more to decouple it from the file watcher completely. Then one would need to always explicitly request file watching - which seems like the natural thing to do.

Reply:
Yeah, that would be nice.